bitnami/etcd/templates/networkpolicy.yaml

```diff
@@ -56,10 +51,9 @@ spec:
             matchLabels:
               {{ template "common.names.fullname" . }}-client: "true"
         - podSelector:
-            matchLabels: {{- include "common.labels.standard" . | nindent 14 }}
-              {{- if .Values.podLabels }}
-              {{- include "common.tplvalues.render" (dict "value" .Values.podLabels "context" $) | nindent 14 }}
-              {{- end }}
+        - podSelector: 
```

Comment:
I think you should remove this line
```suggestion
```

Reply:
done at https://github.com/bitnami/charts/pull/18298/commits/44f2a7821a601f01f0db5381b60f3485771faaf6